CHANGELOG.rst

```diff
@@ -3,19 +3,23 @@ Change Log
 
 ..
    All enhancements and patches to edx_django_utils will be documented
-   in this file.  It adheres to the structure of http://keepachangelog.com/ ,
+   in this file.  It adheres to the structure of https://keepachangelog.com/ ,
    but in reStructuredText instead of Markdown (for ease of incorporation into
    Sphinx documentation and the PyPI description).
 
-   This project adheres to Semantic Versioning (http://semver.org/).
+   This project adheres to Semantic Versioning (https://semver.org/).
 
 .. There should always be an "Unreleased" section for changes pending release.
 
 Unreleased
 ----------
 
-[4.4.2] - 2021-12-23
---------------------
```

Comment:
Should this be restored, and you could add `4.4.3` and update the version?

Reply:
There was never a 4.4.2 tag or release, so I could just restore it. Do we generally mark it in the changelog before it actually exists?

Reply:
I see. Generally, I would add to changelog and update version all in same PR if I plan to release. In this case, looks like `4.4.2`. 

I do not know why `4.4.2` had an entry here. Is the `Updated` section of `4.4.2` just a variation of `4.4.1`, or did additional work happen on the `urls` since `4.4.1`?

Reply:
It looks like someone made a change relating to django.urls and intended to release a 4.4.2 that day, but never got around to it. So this will end up being 4.4.2, in practice.